ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerDemoController.swift

```diff
@@ -336,6 +329,17 @@ class DrawerDemoController: DemoController {
             textField?.resignFirstResponder()
         }
     }
+
+    private var shouldConfirmDrawerDismissal: Bool = false
+    private var expandButton: Button?
```

Comment:
Could we make this a lazy let and create it here instead of in line 126/127?

Reply:
not for button. unfortunately, it is the same democontroller used for different drawercontroller scenarios that we want to create it each time.